force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +103,22 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer result = 0;
+        List<String> charSplit = str.split('');
+        for(String vowelCount : charSplit){
+            if(vowelCount == 'a' || vowelCount == 'e' || vowelCount == 'i' || vowelCount == 'o' || vowelCount == 'u'){
+                result = result + 1;
+            }
+            else{
+                continue;
+            }
+        }
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
-        }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+    
+        
+        return result; // Replace null with the variable you used to store the result
```

Comment:
In my opinion when we are checking for membership like is a vowell. The easiest way and the fastest is to use a Set. 

Here is my solution. 
```
  public static Integer countVowels(String str) {
        // Initialize the result Integer
        Integer vowelCount = 0;
        Set<String> vowels = new Set<String>{
                'a', 'e', 'i', 'o', 'u'
        };
        for (String character : str.split('')) {
            if (vowels.contains(character.toLowerCase())) {
                vowelCount++;
            }
        }
        return vowelCount;
    }
 ```